Foundation/NSURLSession/http/HTTPURLProtocol.swift

```diff
@@ -127,6 +129,7 @@ fileprivate extension _HTTPURLProtocol {
                 set(requestBodyLength: .noBody)
             case (_, .some(let length)):
                 set(requestBodyLength: .length(length))
+                bodyLength = Int64(length)
```

Comment:
@weissi I hope I am getting your comment right here. This length isn't something sent by the server. It corresponds to the length of the data being uploaded - it is either the count of a Data in memory or the length of a file being uploaded (see the definitions of `_HTTPURLProtocol._Body` and `_Body.getBodyLength()`). A size of more that `Int64.max` doesn't seem possible then. Even if it were possible, we'd have an OOM before we get here.